### PR TITLE
Panbha

### PR DESCRIPTION
**Pull request by panbha**
Please review not understanding why so many files in review
I have added only following files

PS I:\git\azure-powershell> git diff --name-status dev..panbha
A       src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureBackupDSCmdletBase.cs
A       src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/BackUp/TriggerBackUp.cs
A       src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/RecoveryPoint/GetAzureBackupRecoveryPoint.cs
A       src/ResourceManager/AzureBackup/Commands.AzureBackup/Models/AzureBackupRecoveryPoint.cs
PS I:\git\azure-powershell>

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@panbha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/AzureBackup/Commands.AzureBackup/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft Azure Powershell")]
+[assembly: AssemblyCompany(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCompany)]
```
Comment (hovsepm) [inline src/ResourceManager/AzureBackup/Commands.AzureBackup/Properties/AssemblyInfo.cs]: Please change WindowsAzure to Azure everywhere in your project

File: src/ResourceManager/AzureBackup/Commands.AzureBackup/Models/AzureBackupItem.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+namespace Microsoft.Azure.Commands.AzureBackup.Cmdlets
+{
+    /// <summary>
+    /// Represents Azure Backup Container
+    /// </summary>
+    public class AzureBackupItem : AzureBackupItemContextObject
+    {
+        /// <summary>
+        /// Status for the Azure Backup Item
```
Comment (hovsepm) [inline src/ResourceManager/AzureBackup/Commands.AzureBackup/Models/AzureBackupItem.cs]: comments over public get/set properties should start with "Gets or sets" text. please fix everywhere in your PR.

File: src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/BackUp/TriggerBackUp.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+                MBS.OperationResponse triggerBackUpInfo =
+                    AzureBackupClient.BackUp.TriggerBackUpAsync(GetCustomRequestHeaders(),
+                    AzureBackupItem.ContainerName,
+                    AzureBackupItem.DataSourceType,
+                    AzureBackupItem.DataSourceId,
+                    CmdletCancellationToken).Result;                
+
+                WriteVerbose("Received policy response");
+                WriteVerbose("Received policy response2");
```
Comment (hovsepm) [inline src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/BackUp/TriggerBackUp.cs]: why do you need to log 4 messages in a row without any actions in between? Please change it to a single meaningfull message.

File: src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/BackUp/TriggerBackUp.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+                WriteVerbose("Converting response");
```
Comment (hovsepm) [inline src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/BackUp/TriggerBackUp.cs]: Why do you need to log 3 messages in a row without any actual action? Please log one meaningful message instead.

File: src/ResourceManager.AzureBackup.sln [hunk 5]
```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Express 2013 for Web
```
Comment (hovsepm) [inline src/ResourceManager.AzureBackup.sln]: remove VS Express line

File: src/AzurePowershell.sln [hunk 6]
```diff
@@ -1,5 +1,5 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
+# Visual Studio Express 2013 for Web
```
Comment (hovsepm) [inline src/AzurePowershell.sln]: remove this line

File: src/AzurePowershell.sln [hunk 7]
```diff
@@ -1,5 +1,5 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
```
Comment (hovsepm) [inline src/AzurePowershell.sln]: revert this line back

**Comment by hovsepm** [comment]
Please make sure you will add scenario tests and mocked records and then send a PR request mail.

**Comment by panbha** [comment]
please ignore pr request

**Comment by panbha** [comment]
do not review. i will undo the pull request

**Comment by ogail** [comment]
@panbha please send new one that's ready to merge

**Pull request closed without merging**